apps/chat/src/store/conversations/conversations.reducers.ts

```diff
@@ -798,7 +800,36 @@ export const conversationsSlice = createSlice({
           .map(({ id }) => id);
       }
     },
+    setAllChosenEmptyFolders: (state) => {
+      if (state.searchTerm) {
+        return state;
+      }
+      state.chosenEmptyFoldersIds = state.folders
+        .filter(
+          (folder) =>
+            !isEntityOrParentsExternal(state, folder, FeatureType.Chat) &&
```

Comment:
By the way...
Is it possible to delete shared folders? Maybe we need to discuss this?

Reply:
It is possible to delete shared folders as well as conversations.
We need to discuss this.

Reply:
we don't allow to multi-select Shared folders at this moment. It was decided on daily meeting